plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHivePageSink.java

```diff
@@ -243,20 +234,22 @@ public static MaterializedResult toMaterializedResult(ConnectorSession session,
         return resultBuilder.build();
     }
 
-    private static ConnectorPageSource createPageSource(HiveTransactionHandle transaction, HiveConfig config, File outputFile)
+    private static ConnectorPageSource createPageSource(TrinoFileSystemFactory fileSystemFactory, HiveTransactionHandle transaction, HiveConfig config, Location location)
+            throws IOException
     {
+        long length = fileSystemFactory.create(ConnectorIdentity.ofUser("test")).newInputFile(location).length();
         Properties splitProperties = new Properties();
-        splitProperties.setProperty(FILE_INPUT_FORMAT, config.getHiveStorageFormat().getInputFormat());
+        splitProperties.setProperty("file.inputformat", config.getHiveStorageFormat().getInputFormat());
```

Comment:
You can use `io.trino.hive.thrift.metastore.hive_metastoreConstants#FILE_INPUT_FORMAT` which is what we use elsewhere in production code